semver/_is_comparator_test.ts

```diff
@@ -2,11 +2,12 @@
 import { assert } from "../assert/mod.ts";
 import { INVALID, MIN } from "./constants.ts";
 import { isComparator } from "./_is_comparator.ts";
+import { formatComparator } from "./_format_comparator.ts";
+import { Comparator } from "./types.ts";
 
 Deno.test({
-  name: "valid_comparator",
+  name: "isComparator()",
   fn: async (t) => {
-    let i = 0;
     const comparators: unknown[] = [
```

Comment:
Perhaps we should assert the correct comparator type instead of the type cast below.

Reply:
Done.